### DISCUSSION TITLE
Health Check Endpoint

### DISCUSSION THREAD

**Discussion by ACPK** [plausible/analytics#1379] [Anything Else]
Does Plausible have an uptime / health check endpoint that is used by https://plausible.instatus.com?

This could be https://plausible.com/health_check or https://plausible.com/health_check?key='password_for_private_health_checks'.

**Comment by Vigasaurus** [comment]
https://plausible.io/api/health

**Reply by ACPK** [reply]
@Vigasaurus Thank you.
